src/test/resources/config/application-artemis.yml

```diff
@@ -34,6 +34,8 @@ artemis:
     version-control:
         user: fake-username-vc # The git user. This is required for build agents
         password: fake-password # The git password. This is required for build agents
+        build-agent-git-username: buildjob_user # Required for https access to localvc
+        build-agent-git-password: buildjob_password # Required for https access to localvc. You can otherwise use a token, or ssh key
```

Comment:
as this is only needed for the localVC & localCI setup I would move this to the localVC/localCI ymls, or, omit it completely as its defined in the buildagent.yml anyway.
Or am I missing something here?

Reply:
This is just a test config file that we use to run the server tests. I don't think it would be necessary to move this to another yml (there aren't even localvc/localci test yml configs). But I actually forgot to add this config to application-localvc.yml since it might be necessary there

Reply:
ah I see :+1: